pkg/sharding/sharding.go

```diff
@@ -87,3 +85,8 @@ func PrependActiveTreeID(uuid string) (FullID, error) {
 	active := dummy.ActiveIndex()
 	return CreateFullID(strconv.FormatUint(active, 10), uuid)
 }
+
+// Returns UUID (with no prepended TreeID) from a UUID or FullID string
+func GetUUIDFromIDString(id string) string {
+	return id[len(id)-UUIDHexStringLen:]
```

Comment:
what happens here if someone passes in a string that isn't a valid ID? you could end up panicking if the lengths are wrong. I think we need some sort of a defensive check here (with potentially an error return value as well)

Reply:
You are right @bobcallaway , sorry I missed it.